development/gcp/cloud-run/github-webhook-gateway/github-webhook-gateway.go

```diff
@@ -0,0 +1,211 @@
+// This program receives Github webhook data and sends it as a pubsub message
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+
+	"github.com/google/go-github/v48/github"
+	"github.com/kyma-project/test-infra/development/gcp/pkg/cloudfunctions"
+	crhttp "github.com/kyma-project/test-infra/development/gcp/pkg/http"
+	"github.com/kyma-project/test-infra/development/gcp/pkg/pubsub"
+	toolsclient "github.com/kyma-project/test-infra/development/github/pkg/client"
+	"github.com/kyma-project/test-infra/development/types"
+)
+
+const (
+	SapToolsGithubURL = "https://github.tools.sap/"
+)
```

Comment:
This const is already defined in a package in our repo. https://github.com/kyma-project/test-infra/blob/fbdfd267f1adedae6c230c006c8f0bf21e23c294/development/github/pkg/client/client.go#L23C50-L23C50

Reply:
This const is not needed anymore, I'll remove it